src/OpenTelemetry.Api/Context/Propagation/CompositePropagator.cs

```diff
@@ -0,0 +1,68 @@
+﻿// <copyright file="CompositePropagator.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace OpenTelemetry.Context.Propagation
+{
+    /// <summary>
+    /// CompositePropagator provides a mechanism for combining multiple propagators into a single one.
+    /// </summary>
+    public class CompositePropagator : ITextFormat
+    {
+        private readonly List<ITextFormat> textFormats;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CompositePropagator"/> class.
+        /// </summary>
+        /// <param name="textFormats">List of <see cref="ITextFormat"/> wire context propagator.</param>
+        public CompositePropagator(List<ITextFormat> textFormats = null)
+        {
+            this.textFormats = textFormats ?? new List<ITextFormat>();
+        }
+
+        /// <inheritdoc/>
+        public ISet<string> Fields => throw new NotImplementedException();
+
+        /// <inheritdoc/>
+        public ActivityContext Extract<T>(ActivityContext activityContext, T carrier, Func<T, string, IEnumerable<string>> getter)
+        {
+            foreach (var textFormat in this.textFormats)
+            {
+                activityContext = textFormat.Extract(activityContext, carrier, getter);
+            }
+
+            return activityContext;
+        }
+
+        /// <inheritdoc/>
+        public void Inject<T>(ActivityContext activityContext, T carrier, Action<T, string, string> setter)
+        {
+            foreach (var textFormat in this.textFormats)
+            {
+                textFormat.Inject(activityContext, carrier, setter);
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool IsInjected<T>(T carrier, Func<T, string, IEnumerable<string>> getter)
+        {
+            throw new NotImplementedException();
```

Comment:
Pretty easy to implement this one. Just do:
```csharp
public bool IsInjected<T>(T carrier, Func<T, string, IEnumerable<string>> getter)
{
            foreach (var textFormat in this.textFormats)
            {
                if (textFormat.IsInjected(carrier, getter))
                {
                     return true;
                }
            }
            return false;
}
```
Without the implementation, users won't be able to plug this in on the HttpClient instrumentation.

This ties into how we have `Extract` working now. As long as one of the child `ITextFormat` has a valid context, it will be extracted.

Reply:
@CodeBlanch , what if: the first one is injected and the second one not. We would return true. Would that be right?

Reply:
what i would think is: if one of the textformats return false, so, false, otherwise, its true. in that case, if the last one is false, it would inject everything again.

Reply:
@eddynaka That works for me (return true only if all are injected). It isn't part of the spec so it's up to us to define it for now.

Reply:
just updated and added some tests to cover both cases